book/src/stake-delegation-and-rewards.md

```diff
@@ -195,3 +200,106 @@ stake.
 ## Example Callflow
 
 <img alt="Passive Staking Callflow" src="img/passive-staking-callflow.svg" class="center"/>
+
+## Staking Rewards
+
+The specific mechanics and rules of the validator rewards regime is outlined
+here.  Rewards are earned by delegating stake to a validator that is voting
+correctly.  Voting incorrectly exposes that validator's stakes to
+[slashing](staking-and-rewardsg.md).
+
+### Basics
+
+The network pays rewards from a portion of network [inflation](inflation.md).
+The number of lamports available to pay rewards for an epoch is fixed and
+must be evenly divided among all staked nodes according to their relative stake
+weight and pariticpation.  The weighting unit is called a
+[point](terminology.md#point).
+
+Rewards for an epoch are not available until the end of that epoch.
+
+At the end of each epoch, the total number of points earned during the epoch is
+summed and used to divide the rewards portion of epoch inflation to arrive at a
+point value.  This value is recorded in the bank in a
+[sysvar](terminology.md#sysvar) that maps epochs to point values.
+
+During redemption, the stake program counts the points earned by the stake for
+each epoch, multiplies that by the epoch's point value, and transfers lamports in
+that amount from a rewards account into the stake and vote accounts according to
+the vote account's commission setting.
+
+### Economics
+
+Point value for an epoch depends on aggregate network participation.  If participation
+in an epoch drops off, point values are higher for those that do participate.
+
+### Earning credits
+
+Validators earn one vote credit for every correct vote that exceeds maximum
+lockout, i.e. every time the validator's vote account retires a slot from its
+lockout list, making that vote a root for the node.
+
+Stakers who have delegated to that validator earn points in proportion to their
+stake.  Points earned is the product of vote credits and stake.
+
+### Stake warmup, cooldown, withdrawal
+
+Stakes, once delegated, do not become effective immediately.  They must first
+pass through a warm up period.  During this period some portion of the stake is
+considered "effective", the rest is considered "activating". Changes occur on
+epoch boundaries.
+
+The stake program limits the rate of change to total network stake, reflected
+in the stake program's `config::warmup_rate` (typically 15% per epoch).
```

Comment:
IIRC epochs are on the order of weeks?  If so, how was 15% determined?

Reply:
8096 slots today, about an hour.  15% was a first try, conservative setting.  now that we have a little faith, we can go right to 1/3

Reply:
got it, thanks